rosetta/templates/rosetta/form.html

```diff
@@ -159,7 +157,7 @@ <h1>{% blocktrans  %}Translate into {{rosetta_i18n_lang_name}}{% endblocktrans %
                         {% if i == page %}
                             <span class="this-page">{{i}}</span>
                         {% else %}
-                            <a href="?page={{i}}{% if query %}&amp;query={{query}}{% endif %}">{{i}}</a>
+                            <a href="?{{ pagination_query_string_base }}&page={{i}}">{{i}}</a>
```

Comment:
Ampersands need to be HTML-encoded

Reply:
Yes they should.

Reply:
I'm working on a minor follow-up PR that will include this fix.